lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -101,6 +100,8 @@ def trigger
       puppet_trigger['start_date'] = self.class.normalized_date("#{trigger['start_year']}-#{trigger['start_month']}-#{trigger['start_day']}")
       puppet_trigger['start_time'] = self.class.normalized_time("#{trigger['start_hour']}:#{trigger['start_minute']}")
       puppet_trigger['enabled']    = trigger['flags'] & Win32::TaskScheduler::TASK_TRIGGER_FLAG_DISABLED == 0
+      puppet_trigger['minutes_interval'] = trigger['minutes_interval'] ||= 0
+      puppet_trigger['minutes_duration'] = trigger['minutes_duration'] ||= 0
```

Comment:
> Disregard the above comment. It looks like the `minutes_duration` of `0` is also reported in the GUI as a `Daily At 1:28 PM every day` trigger.

> So it looks like `1440` and `0` might be interchangeable here.

These are the two different concepts I'm referring to above. `Daily at 1:28 PM` is the trigger, not the repeat interval inside the trigger. For clarity create a weekly trigger and you will see what I mean. It can be a bit confusing. Perhaps this will draw out the clarity some. I redid the image above:

![image](https://cloud.githubusercontent.com/assets/63502/6966755/f4e7749c-d920-11e4-8de8-3f15f619696d.png)